workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
Sorry, this is my fault.

In Java there is no reason for this to work with primitive floats (or normal floats for that matter).
As primitives are always passed by value and the Float objects are immutable, you can not use them as OUT (only IN). This is the normal and expected behaviour, POP-Java should not behave differently.

In which code do you need them? can you point me to the file on github so i can see why it would be required there?